### PR TITLE
(PUP-8315) add translations decorations to Puppet.info calls

### PR DESCRIPTION
**Pull request by ericldelaney**

### CODE REVIEW

**Comment by kris-bosland** [comment]
Looks good.

File: lib/puppet/indirector/file_bucket_file/file.rb [hunk 1]
```diff
@@ -158,7 +158,7 @@ def save_to_disk(bucket_file, files_original_path, contents_file, paths_file)
         Puppet::FileSystem.exclusive_open(paths_file, 0640, 'a+:external') do |f|
           if Puppet::FileSystem.exist?(contents_file)
             if verify_identical_file(contents_file, bucket_file)
-              Puppet.info "FileBucket got a duplicate file #{bucket_file.checksum}"
+              Puppet.info _("FileBucket got a duplicate file %{file_checksum}") % { file_checksum: bucket_file.checksum }
```
Comment (melissa) [inline lib/puppet/indirector/file_bucket_file/file.rb]: Is `FileBucket` the name of a method here? Or is this something that should be translated?

Comment (ericldelaney) [reply lib/puppet/indirector/file_bucket_file/file.rb]: Yeah, I thought I added a comment for that. Let me fix it.

**Comment by ericldelaney** [comment]
@melissa Updated.

**Pull request merged**